### DISCUSSION TITLE
代理显示可使用但无法播放歌曲

### DISCUSSION THREAD

**Discussion by NimaQu** [1715173329/UnblockNeteaseMusic#394] [Q&A] [answered]
网易云版本最新，项目代码最新
![image](https://user-images.githubusercontent.com/26685881/146419802-bb103ae8-d1b6-4bfe-82ad-a00b3cfb3089.png)
![image](https://user-images.githubusercontent.com/26685881/146419552-9f33e949-4381-4184-8bd0-f9cf86c8d638.png)
![image](https://user-images.githubusercontent.com/26685881/146419623-2c1c9267-8970-4960-8a24-7b463a8a8ea6.png)

**Comment by NimaQu** [comment]
update: 直接从 release 页面下载的 exe 没问题

**Comment by 1715173329** [comment, marked as answer]
如果劫持有效，测试将显示“该代理不可使用”
您可能需要监听 https 端口以正常使用
e.g. `node app.js -p 11451:11452`

**Comment by GoldJinJin** [comment]
我也出现同样的问题，下载最新的exe还是不行

**Reply by GoldJinJin** [reply]
问题已经解决，首先我使用的是exe 然后突然发现出现上述问题，后来发现是8080端口好像和一个应用冲突了，换成注册windows服务方式 更换配置文件中的端口号解决问题
